### ISSUE TITLE
Request for Chaturbate support

### ISSUE THREAD

**Issue by Daisenki** [freaktechnik/justintv-stream-notifications#355]
Any plans for new platforms? Chaturbate would be cool ;)

**Comment by freaktechnik** [comment]
I won't add such a provider until the parental controls have been re-added.

Not looking good for APIs, either way. I could find some php lib for affiliates, but that's likely not to help: https://github.com/hillipino/Chaturbate-API. Maybe I can just use their feed, not sure how reliable it is: https://chaturbate.com/feed/latest/

**Comment by freaktechnik** [comment]
I guess there would also have to be a setting to toggle nsfw thumbnails.

**Comment by Daisenki** [comment]
To be honest, I don't see any different between mature content on Twitch and porn on Chaturbate, both is NSFW and adult only stuff.

**Comment by freaktechnik** [comment]
Right, and there already is a generic NSFW filter built into the extension, it just hadn't occurred to me before this request that a thumbnail filter should be an option.

**Comment by freaktechnik** [comment]
(I've spun the thumbnail thing off as #356)
